components/automate-cli/pkg/verifyserver/services/batchcheckservice/batchcheckservice_test.go

```diff
@@ -1,36 +1,181 @@
 package batchcheckservice
 
 import (
+	"bytes"
 	"encoding/json"
-	"testing"
 
-	"github.com/gofiber/fiber/v2"
+	"errors"
+	"io/ioutil"
+	"net/http"
+	"testing"
 
 	"github.com/chef/automate/components/automate-cli/pkg/verifyserver/constants"
 	"github.com/chef/automate/components/automate-cli/pkg/verifyserver/models"
 	"github.com/chef/automate/components/automate-cli/pkg/verifyserver/services/batchcheckservice/trigger"
+	"github.com/chef/automate/components/automate-cli/pkg/verifyserver/utils/httputils"
+	"github.com/chef/automate/lib/io/fileutils"
+	"github.com/chef/automate/lib/logger"
+	"github.com/gofiber/fiber/v2"
+
 	"github.com/stretchr/testify/assert"
 )
 
+func SetupMockHttpRequestClient(responseJson string, err error, shouldStartMockServerOnSomeNodesOnly bool) httputils.IHttpRequestClient {
+	r := ioutil.NopCloser(bytes.NewReader([]byte(responseJson)))
```

Comment:
```suggestion
	r := io.NopCloser(bytes.NewReader([]byte(responseJson)))
```
ioutil.NopCloser is deprecated

Reply:
Done